buildbot/Makefile

```diff
@@ -1,43 +1,39 @@
-RIAK_CONF = ${RIAK_DIR}/etc/riak.conf
-# ADVANCED_CONF = ${RIAK_DIR}/etc/advanced.config
-# RIAK = ${RIAK_DIR}/bin/riak
-RIAK_ADMIN = ${RIAK_DIR}/bin/riak-admin
-CERTS_DIR = $(shell pwd)/../riak/tests/resources
+ifndef RIAK_DIR
+$(error RIAK_DIR is not set)
+endif
+
+RIAK_CONF = $(RIAK_DIR)/etc/riak.conf
+RIAK_ADMIN = $(RIAK_DIR)/bin/riak-admin
+CERTS_DIR = $(realpath $(CURDIR))/../riak/tests/resources
```

Comment:
Not sure `realpath` is installed on all platforms by default.  On CentOS 6, for example:
```shell
$ realpath
-bash: realpath: command not found
```